client/components/homepage/testimonials/Testimonials.jsx

```diff
@@ -0,0 +1,41 @@
+import React from 'react';
+import Testimonial from './Testimonial';
+import testimonialInfo from './testimonialInfo';
+
+
+const Testimonials = () => {
+  const settings = {
+    dots: true,
+    infinite: true,
+    slidesToShow: 1,
+    slidesToScroll: 1,
+    autoplay: true,
+    nextArrow: false,
+    prevArrow: false,
+    autoplaySpeed: 2000
+  };
+
+  return (
+    <section id="testimonials" className="section testimonials background-gray-lightest">
+      <div className="container">
+        <h2 className="text-center">
+          What the world is saying about Ribbit
```

Comment:
Personal preference, feels like this looks a bit long and drawn out. Would prefer shorter wording.